docs/ar/week03/03.md

```diff
@@ -1,17 +1,16 @@
-| lang-ref | lang | title | translator |
-| ------ | ------ | ------ | ------ |
-| ch.03 | ar | الأسبوع 3 | [aissam out](https://github.com/aissam-out) |
-
-<div dir="rtl">
+---
+lang-ref: ch.03
+title: Week 3
+---
 
 <!-- Lecture part A
 We first see a visualization of a 6-layer neural network. Next we begin with the topic of Convolutions and Convolution Neural Networks (CNN). We review several types of parameter transformations in the context of CNNs and introduce the idea of a kernel, which is used to learn features in a hierarchical manner. Thereby allowing us to classify our input data which is the basic idea motivating the use of CNNs.-->
 
-## محاضرة الجزء أ
+## محاضرة الجزء A

```

Comment:
I know I've asked for this, but now I'm not too sure it's a good idea. See [this](https://dlsquad.slack.com/archives/C0193CG5VNX/p1598772792300600).
Any thoughts?

Reply:
Yeah ! I think it's better to use the arabic letters .. I made the changes you requested :) 👍